student/src/main/java/tds/student/web/StudentContext.java

```diff
@@ -164,12 +164,19 @@ public static void saveOppInfo (String testKey, String testID, OpportunityInfo o
     if (oppInfo.getBrowserKey() != null) {
       StudentCookie.setCookieData ("O_BKEY", oppInfo.getBrowserKey ().toString ());
     }
-
-    TDSIdentity.getCurrentTDSIdentity().setAuthCookieValue(EXAM_CLIENT_NAME, oppInfo.getExamClientName());
-    TDSIdentity.getCurrentTDSIdentity().setAuthCookieValue(EXAM_ID_COOKIE_KEY, oppInfo.getExamId().toString());
+    if(oppInfo.getExamClientName() != null) {
```

Comment:
@jtreuting, @hbgwhite and I discussed this change.  Right now this blows up if you are running with the `remoteEnabled` flag turned off.  Other areas of the application will throw if remote is on and these aren't populated which I think is fine.  We can probably remove the null checks once we turn off the legacy for good.